src/dbally/similarity/chroma_store.py

```diff
@@ -0,0 +1,96 @@
+from typing import List, Literal, Optional, Union
+
+import chromadb
+
+from dbally.embedding_client.base import EmbeddingClient
+from dbally.similarity.store import SimilarityStore
+
+
+class ChromadbStore(SimilarityStore):
+    """Class that stores text embeddings using [Chroma](https://docs.trychroma.com/)"""
+
+    def __init__(
+        self,
+        index_name: str,
+        client: chromadb.Client,
+        embedding_calculator: Union[EmbeddingClient, chromadb.EmbeddingFunction],
+        max_distance: Optional[float] = None,
+        distance_method: Literal["l2", "ip", "cosine"] = "l2",
+    ):
+        super().__init__()
+        self.index_name = index_name
+        self.client = client
+        self.embedding_calculator = embedding_calculator
+        self.max_distance = max_distance
+
+        self._metadata = {"hnsw:space": distance_method}
+
+    def _get_collection(self) -> chromadb.Collection:
```

Comment:
The name "collection" here clashes with dbally's Collection. It would definitely try to be more explicit about this being chroma's collection - both in naming things in code and in docstrings.